active/af3-resynthesize-phi.py

```diff
@@ -159,11 +166,16 @@ def _get_annotations_by_id_and_created_date(blobid, date):
                       "WHERE date_created = %s and hdcorcablobid = %s "
                       "AND (category = %s "
                       "     OR category = %s)".format(table=common_variables.ANNOTATION_TABLE))
-
-    return common_hooks.ANNOTATIONS_DB.get_records(src_select_stmt, parameters=(
+    try:
+      return common_hooks.ANNOTATIONS_DB.get_records(src_select_stmt, parameters=(
                                              date, blobid, common_variables.REVIEW_BYPASSED_ANNOTATION_TYPE,
                                              common_variables.BRAT_REVIEWED_ANNOTATION_TYPE))
-
+    except OperationalError as e:
+        message = ("An OperationalError occured while trying to fetch annotations from the source data server"
+                   " for blobid {}".format(blobid))
+        print(message)
+        common_functions.log_error_and_failure_for_resynth_note_job(run_id, blobid, hdcpupdatedate, message,
+                                                                  "Get individual note annotations")
```

Comment:
Do we need to return something (eg: None) at this point? or raise the operational error and let the caller figure it out?

Reply:
yeah, I'll add a return None

Reply:
Do we want to retry if there is an OperationalError? A lot of the times it runs thru when you rerun the dag.

Reply:
but if it doesn't pass on another attempt than we're back where we started and not moving onto the next task